### PR TITLE
Editorial and quality assurance on council.html

### PR DESCRIPTION
**Pull request by koalie**
... including re-using terminology from the Process Document where relevant to avoid mis-interpretation.

### CODE REVIEW

File: council/council.html [hunk 1]
```diff
@@ -70,8 +70,7 @@ <h2 id="council"><a href="#council">Cases that warrant a Council</a></h2>
     that require senior review. Some topics do not lend themselves to compromise, they are inherently yes/no decisions. It could be hard to find common ground.
   </p>
   
-  <p>Nonetheless, the fact that the process is oriented towards consensus implies that before an objection gets a senior review there already has been a great
-    deal of work on the issue. There is generally clarity on what the W3C decision was (that is being objected to). 
+  <p>Nonetheless, the fact that the process is oriented towards consensus implies that before an objection gets a senior review there already has been significan work on the issue. There is generally clarity on what the W3C decision was (that is being objected to).
```
Comment (ylafon) [inline council/council.html:73]: significan -> significant

File: council/council.html [hunk 2]
```diff
@@ -167,7 +166,7 @@ <h2 id="referral"><a href="#referral">Referral to W3C Council</a></h2>
     <p>The report should be available for review by the relevant groups and objectors.</p>
     <ul>
       <li><p>If any participant feels that their viewpoint is not fairly represented, a limited time effort should be expended to get all relevant information into the document.</p></li>
-      <li><p>If at the end of the limited time effort the Team determines that it is not possible to get agreement on the summary document, then the Team publishes
+      <li><p>If at the end of the alloted time the Team determines that it is not possible to get agreement on the summary document, then the Team publishes
```
Comment (ylafon) [inline council/council.html:169]: alloted -> allotted (or allocated?)

File: council/council.html [hunk 3]
```diff
@@ -183,11 +182,11 @@ <h2 id="referral"><a href="#referral">Referral to W3C Council</a></h2>
     </ul>
   </li>
   <li>
-    <p>In extraordinary circumstances, the Council may vote to <a class="TODO" href="https://www.w3.org/Consortium/Process/Drafts/#council-delegation">delegate</a> or
+    <p>In exceptional circumstances, the Council may vote to <a class="TODO" href="https://www.w3.org/Consortium/Process/Drafts/#council-delegation">delegate</a> or
```
Comment (ylafon) [inline council/council.html:185]: Process says
> In extraordinary cases

So better to keep `extraordinary` here

File: council/council.html [hunk 4]
```diff
@@ -227,16 +226,16 @@ <h3 id="convincing-the-council"><a href="#convincing-the-council">II. Convincing
   supported by all is always preferred. Nothing in this discussion precludes such an approach or forces an adversarial judicial proceeding.
   Nonetheless, there should be a clear expectation about the starting point for Council deliberations.</p>
 
-<p>Objectors should not be looking for the Council to come up with new arguments to support their point of view.
+<p>Objectors should not be looking at the Council to come up with new arguments to support their point of view.
```
Comment (ylafon) [inline council/council.html:229]: `for` seems better here, but native speakers should confirm

**Review by ylafon — CHANGES_REQUESTED** [review]
Looks good, thanks!
Some minor details/clarifications inline requiring changes

File: council/council.html (hunk 2 above)
Comment (koalie) [reply council/council.html:169]: "allocated" is indeed much better. +1

File: council/council.html (hunk 3 above)
Comment (koalie) [reply council/council.html:185]: I suggested "exception" to match the note in https://www.w3.org/2023/Process-20230612/#council-short-circuit which is what this line is about.

**Comment by koalie** [comment]
Thanks for the review on the review, Yves!

File: council/council.html (hunk 3 above)
Comment (ylafon) [reply council/council.html:185]: This is about [Extraordinary Delegation](https://www.w3.org/2023/Process-20230612/#council-delegation), so it makes sense to keep the same adjective, but both work.

Comment (koalie) [reply council/council.html:185]: Ah you're right. Whichever you prefer, then!

**Review by tantek — CHANGES_REQUESTED** [review]
These improvements look great overall. Thanks @koalie.

One minor nit in council.html which I think could be improved for readability/grammar:
> there already has been significan work on the issue

should be:
> there has already been significant work on the issue

File: council/council.html (hunk 1 above)
Comment (plehegar) [inline council/council.html:73]:
```suggestion
  <p>Nonetheless, the fact that the process is oriented towards consensus implies that before an objection gets a senior review there already has been significant work on the issue. There is generally clarity on what the W3C decision was (that is being objected to). 
```

File: council/council.html (hunk 4 above)
Comment (plehegar) [reply council/council.html:229]: look at is correct.

**Review by ylafon — APPROVED** [review]

**Pull request merged**